src/languages/en.ts

```diff
@@ -557,19 +557,19 @@ export default {
         splitAmount: ({amount}: SplitAmountParams) => `split ${amount}`,
         didSplitAmount: ({formattedAmount, comment}: DidSplitAmountMessageParams) => `split ${formattedAmount}${comment ? ` for ${comment}` : ''}`,
         amountEach: ({amount}: AmountEachParams) => `${amount} each`,
-        payerOwesAmount: ({payer, amount}: PayerOwesAmountParams) => `${payer} owes ${amount}`,
+        payerOwesAmount: ({payer, amount}: PayerOwesAmountParams) => `${payer ? `${payer} ` : ''}owes ${amount}`,
```

Comment:
WDYT about adding a function that check if the attribute exists and returns it or empty string instead of writing similar logic for each case?
e.g.`getTranslationString = str => str || ''

Reply:
I'd rather go with Kuba's solution 😃

Reply:
ah, but no - here I'm adding an additional space after payer (I don't want it when it's no payer)

and with creating a function I will still have to check for empty string to add/not to add this additional space
-> kinda similar to the code 2 lines above `split ${formattedAmount}${comment ? for ${comment} : ''},`